### PR TITLE
Library: Add aXe tests for media library elements

### LINKED ISSUE #3524: Add jest-axe tests for all media gallery components
See #3398 for example

### PR DESCRIPTION
**Pull request by samwhale**
## Context

Add jest-axe tests to media.

## Summary

Added jest axe tests.

## Relevant Technical Choices

Didn't see any jest-axe tests for elements in the media3p panel so I added some. 

## To-do

n/a

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #3524

### CODE REVIEW

File: packages/story-editor/src/components/library/panes/media/media3p/test/accesibility.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        height: 353,
+      },
+    },
+    src: 'http://lala.com',
+    type: 'image',
+    width: 530,
+  },
+];
+
+describe('automated accessibility tests', () => {
```
Comment (samwhale) [inline packages/story-editor/src/components/library/panes/media/media3p/test/accesibility.js:64]: Did not see any axe tests for the media3p panel (like #3398 was referring to). Seems like #3398 was closed without these tests being added?

Comment (swissspidy) [reply packages/story-editor/src/components/library/panes/media/media3p/test/accesibility.js:64]: I think #3398 was just about fixing a11y issues, and #3524 was opened for adding aXe tests specifically.

File: packages/story-editor/src/components/library/panes/media/media3p/test/accesibility.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  it('should render Media3pIcon without accessibility violations', async () => {
+    const { container } = renderWithTheme(<Media3pIcon />);
+
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+  });
+
+  it('should render XXXXXx without accessibility violations', async () => {
```
Comment (samwhale) [inline packages/story-editor/src/components/library/panes/media/media3p/test/accesibility.js]:
```suggestion
  it('should render ProviderTab without accessibility violations', async () => {
```

**Comment by spacedmonkey** [comment]
The AXE tests are also run on E2E tests as well. We currently skip lots of AXE checks. See 

https://github.com/google/web-stories-wp/blob/539cbe1391e51183931bbe98a53dde5892564b3f/packages/e2e-tests/src/config/bootstrap.js#L221-L241

I wonder if any of these can be unskipped because of this change.

**Review by BrittanyIRL — COMMENTED** [review]
looks like ya got some conflicts, but this is exciting to have added in. Also curious if this eliminates any of the disabled rules Jonny pointed out in the e2e tests.

**Comment by samwhale** [comment]
@spacedmonkey @BrittanyIRL removed the rules that didn't need to be disabled anymore 🥳

**Review by maxyinger — APPROVED** [review]

**Review by BrittanyIRL — APPROVED** [review]
So satisfying to be able to enable these rules! 💯

**Pull request merged**